README.md

```diff
@@ -129,18 +205,14 @@ For a code resource, please see the [examples](./examples/airgap/main.go).
 
 ### Setup
 
-You need:
-  * go >= 1.14
-  * rust >= 1.41.0
-  * openapi-generator To re-generate rpc scaffold ([install link](https://openapi-generator.tech))
-  * golangci To run linter (check https://github.com/golangci/golangci-lint#install )
+In addition to the dependencies listed above under the instructions for running from `rosetta` source code, you also need:
+
+* openapi-generator To re-generate rpc scaffold ([install link](https://openapi-generator.tech))
```

Comment:
nit: wrap `openapi-generator` with code formatting

Reply:
Does it matter what version?

Reply:
tbh less familiar with this part, a holdover from the previous README (want to keep it around unless we know we can delete). ✅ added formatting